src/test/javascript/spec/component/exercises/shared/exercise-scores.component.spec.ts

```diff
@@ -106,14 +96,14 @@ describe('Exercise Scores Component', () => {
                 MockHasAnyAuthorityDirective,
                 MockPipe(ArtemisTranslatePipe),
                 MockPipe(ArtemisDatePipe),
+                MockDirective(NgModel),
             ],
             providers: [
-                { provide: TranslateService, useClass: MockTranslateService },
                 { provide: ActivatedRoute, useValue: route },
-                { provide: AlertService, useClass: MockAlertService },
-                { provide: Router, useValue: router },
                 { provide: LocalStorageService, useClass: MockSyncStorage },
                 { provide: SessionStorageService, useClass: MockSyncStorage },
```

Comment:
You could try to instead mock the Profile-, CourseManagement and ProgrammingSubmissionService. Or did you try that already and it did not work?

Reply:
Changed in 28b2984, works fine.